app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,76 @@
+class CommentsController < ApplicationController
```

Comment:
I swear I look up singular/plural every time I generate a controller. Sorry about that.